dashboard/app/views/weblab_host/index.html.haml

```diff
@@ -0,0 +1,6 @@
+%html{spellcheck: 'false'}
+  %head
+  %body{style: 'margin: 0'}
+    %div{style: 'height: 20px'}
+    #bramble{style: 'height: 100vh;overflow: hidden;'}
+    %script{src: '../blockly/js/bramble/thirdparty/require.min.js', 'data-main':'../blockly/media/weblab/bramblehost.js'}
```

Comment:
We should probably use [`asset_path` or `minifiable_asset_path`](https://github.com/code-dot-org/code-dot-org/blob/ef9aaa7386de016ed9b484daa057f5a22a24e87c/dashboard/app/views/levels/_apps_dependencies.html.haml#L24) here.